src/Data/Form.purs

```diff
@@ -185,69 +185,73 @@ instance decodeInputSource :: DecodeJson InputSource where
 
 -- Test
 
-testPageEither :: Either String (Page Expr)
-testPageEither = decodeJson =<< jsonParser testPageJson
-
-testPageJson :: String
-testPageJson = """
-  { "name": "Profile"
-  , "contents":
-    [ """ <> testSection <> """
+testPage :: Page Expr
+testPage =
+  { name: "Profile"
+  , contents:
+    [ testSection
     ]
   }
-"""
-
-testSection :: String
-testSection = """
-  { "name": "Name"
-  , "contents":
-    [ """ <> firstName <> """
-    , """ <> lastName <> """
-    , """ <> active <> """
+
+testSection :: Section Expr
+testSection =
+  { name: "Name"
+  , contents:
+    [ firstName
+    , lastName
+    , active
     ]
   }
-"""
-
-testField :: String -> String -> String -> String
-testField n d i = """
-  { "name": """ <> val n "String" <> """
-  , "visibility": """ <> val true "Boolean" <> """
-  , "description": """ <> val d "String" <> """
-  , "key": "firstName"
-  , "input": """ <> i <> """
-  }
-"""
-
-textInput :: String
-textInput = """
-  { "type": "Text"
-  , "required": """ <> val true "Boolean" <> """
-  , "placeholder": """ <> val "" "String" <> """
-  , "value": { "value": null, "source": null }
-  , "default": null
-  , "maxLength": null
-  , "minLength": null
-  }
-"""
 
-toggleInput :: String
-toggleInput = """
-  { "type": "Toggle"
-  , "value": { "value": null, "source": null }
-  , "default": """ <> val false "Boolean" <> """
+firstName :: Field Expr
+firstName =
+  { name: mkExpr (Val identity "First Name")
```

Comment:
I know this test data isn't meant to stick around, but while we're manually constructing these, it might be worthwhile adding a helper function so we can just do something like
```purescript
  name: val_ "First Name"
```

Reply:
Good idea! I went ahead and made a helper for all the cases since we're most likely going to be writing them once we build momentum.